integration-tests/narayana-lra/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-narayana-lra</artifactId>
+    <name>Quarkus - Integration Tests - Narayana LRA Coordinator and Participant</name>
+
+    <properties>
+        <quarkus.http.test-port>8081</quarkus.http.test-port>
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <!-- Minimal test dependencies to *-deployment artifacts for consistent build order -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-narayana-lra-deployment</artifactId>
+            <version>${project.version}</version>
+            <type>pom</type>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-deployment</artifactId>
```

Comment:
I would test with classic resteasy but also with resteasy reactive. Should not change anything, but we never know.

Reply:
I will give it another go. I did try modifying the dependencies and got errors like:
```
Caused by: java.lang.IllegalStateException: Please make sure there is only one provider of the following capabilities:
capability io.quarkus.rest is provided by:
  - io.quarkus:quarkus-resteasy-reactive::jar:999-SNAPSHOT
  - io.quarkus:quarkus-resteasy::jar:999-SNAPSHOT
```
I couldn't figure who was still pulling the non reactive version but I will investigate further.

Reply:
See my earlier comment above (https://github.com/quarkusio/quarkus/pull/17903#discussion_r663043617)

Reply:
You need to remove `io.quarkus:quarkus-resteasy`, both can't be used at the same time.

Reply:
That is what I did.

Reply:
Hum, that's kind of annoying. Does your implementation pull the classic Resteasy. We would need to have a look.